include/affinity.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <assert.h>
+
+typedef uint64_t uint64;
+typedef uint32_t uint32;
+
+#define MAX_CORES ((uint64)2048)
+#define AFFINITY_MASK_BITS_PER_QUAD ((uint64)64)
+#define AFFINTY_MASK_NUM_QUADS (MAX_CORES / AFFINITY_MASK_BITS_PER_QUAD)
```

Comment:
Typo in `AFFINTY_MASK_NUM_QUADS`, should be `AFFINITY_MASK_NUM_QUADS`

Also why are these needed? They are currently unused, since `_affinity_mask` is unused. But even if they were used, is it really necessary to hardcode the maximum core count? Shouldn't these be provided by the platform specific implementation used?